README/ReleaseNotes/v638/index.md

```diff
@@ -65,6 +65,8 @@ The following people have contributed to this new version:
 ### Minuit2
 
 * Behavior change: building ROOT using `minuit2_omp=ON` option no longer enables OpenMP parallelization by default. One has to call now additionally GradientCalculator::SetParallelOMP().
+* The functionality of the `FCNGradAdapter` got absorbed into the `FCNAdapter`. This also meanst that the `FCNGradAdapter.h` header is gone.
+* The `FCNAdapter` is not a templated class anymore, but instead uses `std::function` to wrap the functions. Please remove the constructor template argument.
```

Comment:
I am not sure I understand the call for action :( [I am probably just missing the actual context in which this is used]

Reply:
You're right. Since the class is probably not present in user code, the call to action is superfluous